Cannot find module 'import-sort-style-module-alias-x'

Hi, I have a fresh install of my project on a new machine and suddenly import-sort stopped working. In the output log I see this:

```bash
4/13/2021, 3:45:10 PM:
----------------------
Cannot find module 'import-sort-style-module-alias-x'
Require stack:
- /Users/danielkoprowski/.vscode/extensions/amatiasq.sort-imports-6.2.4/out/config-cache.js
- /Users/danielkoprowski/.vscode/extensions/amatiasq.sort-imports-6.2.4/out/sort.js
- /Users/danielkoprowski/.vscode/extensions/amatiasq.sort-imports-6.2.4/out/extension.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/loader.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-fork.js
```

This is my `importSort` in `package.json`.

```json
  "importSort": {
    ".js, .jsx, .es6, .es": {
      "parser": "babylon",
      "style": "module-alias",
      "options": {
        "alias": [
          "~~test",
          "~assets",
          "~components",
          "~config",
          "~consts",
          "~ducks",
          "~helpers",
          "~hooks",
          "~routes",
          "~scenes",
          "~services"
        ],
        "overrideBuiltinModules": true
      }
    },
    ".ts, .tsx": {
      "parser": "typescript",
      "style": "module-alias",
      "options": {
        "alias": [
          "~~test",
          "~assets",
          "~components",
          "~config",
          "~consts",
          "~ducks",
          "~helpers",
          "~hooks",
          "~routes",
          "~scenes",
          "~services"
        ]
      }
    }
  }
```

Everything is freshly installed. Strange that it doesn't work out-of-the-box. Do you have any ideas?

---
## VS Code extension
```
sort-imports amatiasq.sort-imports VSC Sort Import
v6.2.4
```

And packages:
```
    "import-sort-parser-typescript": "^6.0.0",
    "import-sort-style": "^6.0.0",
    "import-sort-style-module-alias": "^1.1.0",
```
---
## CLI
It works well from CLI:

```
npx import-sort-cli [FILE]
```

It worked well so it seems like a bug in the extension.

---

### Previously I've created an issue in "import-sort-style-module-alias" repo.

https://github.com/fsmaia/import-sort-style-module-alias/issues/9#

--- a/src/config-cache.ts
+++ b/src/config-cache.ts
@@ -60,16 +60,19 @@ export function getConfig(document: TextDocument): IResolvedConfig {
   const config = clone(DEFAULT_CONFIGS);
   const defaultSortStyle = getConfiguration<string>('default-sort-style');
 
-  Object.keys(config).forEach((key) => {
-    config[key].style = require.resolve(
-      `import-sort-style-${defaultSortStyle}`
-    );
-  });
-
   // Initialize reference to the document current workspace folder
   if (!currentWorkspaceFolder) {
     currentWorkspaceFolder = workspace.getWorkspaceFolder(document.uri);
   }
+
+  Object.keys(config).forEach((key) => {
+    config[key].style = require.resolve(
+      `import-sort-style-${defaultSortStyle}`,
+      {
+        paths: [currentWorkspaceFolder.uri.path]
+      }
+    );
+  });
 
   if (!useCache || !cachedConfig || hasWorkspaceFolderChanged(document)) {
     cachedConfig = sortGetConfig(

